src/filter/GaussianFilter.js

```diff
@@ -0,0 +1,58 @@
+import IJ from '../ij';
+
+export default function gaussian_filter(n, sigma){
+	var cols = this.width;
+	var new_img = this.clone();
+	var kernel = get_gaussian_kernel(n, sigma);
+	var temp = cols*n; //variable for checking the image border
+	for(var p = cols*n+n; p < this.data.length - cols*n; p++){
+		if(p == temp+cols-n){
+			p += 2*n - 1;
+		}else{
+			new_img.data[p] = convolution(p, this.data, kernel);
+		}
+	}
+}
+
+function get_gaussian_kernel(n, sigma){
```

Comment:
In javascript the name of a function is written using camelCase. In this case it should be getGaussianKernel